test/verify/check-accounts

```diff
@@ -187,6 +187,7 @@ class TestAccounts(MachineCase):
         b.logout()
         self.allow_restart_journal_messages()
         b.open("/users")
+        b.wait_visible("#login")
```

Comment:
It feels a  bit sad having to to this -- `set_val()` now already waits for its selector to become visible. What difference does it make to wait for `#login` here? I. e. how does the test fail without this?

Reply:
Yes, agree, but without this it was failing on all instances where tests were verifying that some error message pops up. I checked the code and there are some checks that verify some elements exist and then show error message (I guees to work around different types of verification). Without this wait no error messages were shown. Possibly this can be done differently but this was the only thing that really helped with the timing (sleep(1) does not count)

Reply:
Ack, let's figure this out in a follow-up then.